src/main/java/haste/commands/AddCommand.java

```diff
@@ -0,0 +1,81 @@
+package haste.commands;
+
+import haste.data.TaskList;
+import haste.tasks.Deadline;
+import haste.tasks.Event;
+import haste.tasks.Task;
+import haste.tasks.ToDo;
+import haste.ui.Ui;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a command that adds tasks to a TaskList.
+ */
+public class AddCommand extends Command {
+
+    private String desc;
+    private LocalDateTime start;
+    private LocalDateTime end;
+    private String type;
```

Comment:
Perhaps you can use a more specific name for this variable? (eg. taskType)

Reply:
Yeap I should have put a more descriptive name for variables like desc and type